lib/fluent/event.rb

```diff
@@ -21,6 +21,10 @@ class EventStream
     include Enumerable
     include MessagePackFactory::Mixin
 
+    def records
```

Comment:
I'll create another pull-request to change `Chunk#records` and `EventStream#records` to `#num_events` later.
Let me merge this right now.